packages/multi-entry/src/index.js

```diff
@@ -1,56 +1,77 @@
-/* eslint-disable consistent-return, no-param-reassign */
+/* eslint-disable no-param-reassign */
 
-import matched from 'matched';
+import virtual from '@rollup/plugin-virtual';
+import { promise as matched } from 'matched';
 
-const entry = '\0rollup:plugin-multi-entry:entry-point';
+const DEFAULT_OUTPUT = 'multi-entry.js';
+const AS_IMPORT = 'import';
+const AS_EXPORT = 'export * from';
 
-export default function multiEntry(conf) {
-  let include = [];
-  let exclude = [];
-  let exporter = (path) => `export * from ${JSON.stringify(path)};`;
+export default function multiEntry(conf = {}) {
+  const config = {
+    include: [],
+    exclude: [],
+    entryFileName: DEFAULT_OUTPUT,
+    exports: true,
+    ...conf
+  };
+
+  let prefix = config.exports === false ? AS_IMPORT : AS_EXPORT;
```

Comment:
Seems minor, but checking to make sure this isn't hiding a bug: you're always setting config.exports to true immediately before this condition.

Reply:
The `...conf` at the end of the config definition will overwrite the default `exports: true`, since the documentation describes that you can pass a `multi({ exports: false })` object into the plugin itself. Therefore, given that is a means by which you can configure this plugin, I have to initialise the prefix with the potential of being either an import/export type before I process the input. The style is the same as `Object.assign({ exports: true }, conf)`

That is how I understand the code when I came to have a bash at it.

Reply:
Whoops, of course you are right. Thanks!